### PR TITLE
RSDK-7438: Remove incorrect assertion in `TestSessionsMixedOwnersImplicitAuth`.

### PR DESCRIPTION
**Pull request by dgottlieb**

### CODE REVIEW

File: robot/session_test.go [hunk 1]
```diff
@@ -757,11 +758,20 @@ func TestSessionsMixedOwnersImplicitAuth(t *testing.T) {
 	}))
 	test.That(t, err, test.ShouldBeNil)
 
+	// This test sets up two clients to the same motor. The test sets up this (contrived) scenario:
+	// 1) Client 1 will first `SetoPwer` with a session heartbeat. This operation returns with the
```
Comment (cheukt) [inline robot/session_test.go:762]: SetoPwer -> SetPower

File: robot/session_test.go [hunk 2]
```diff
@@ -786,16 +796,17 @@ func TestSessionsMixedOwnersImplicitAuth(t *testing.T) {
 	test.That(t, resp.Id, test.ShouldNotEqual, sessID)
 	test.That(t, resp.Id, test.ShouldNotEqual, "")
 
-	// this is the only one heartbeating so we expect a stop
+	// Assert that closing `roboClient1` results in heartbeats stopping that propagates to `Stop`ing
```
Comment (cheukt) [inline robot/session_test.go:799]: roboClient1 -> robotClient1

**Review by cheukt — COMMENTED** [review]
should we replace the other instances of the panic in this test file as well? Think there are 3

File: robot/session_test.go (hunk 2 above)
Comment (dgottlieb) [reply robot/session_test.go:799]: The variable name here is `roboClient1`. Though I agree the variable name of `robotClient1` would have been a bit more conventional.

**Comment by dgottlieb** [comment]
> should we replace the other instances of the panic in this test file as well? Think there are 3. Otherwise, looks great and the comments help a lot!

Didn't even see those, thanks for calling it out!

I did leave one of the [panics alone](https://github.com/viamrobotics/rdk/pull/3868/files#diff-f8b1ef8fdc233258a7ef17977ebf1dc2ecb5b4d2d100497452dfc8a57fa169b1R598) (it wasn't exactly the same). That assertion is still technically wrong, but we can take a look at options/evaluate the risk of removing coverage when/if that fires.

**Review by cheukt — APPROVED** [review]
LGTM as long as tests pass

**Pull request merged**